src/SDKs/Search/DataPlane/Microsoft.Azure.Search.Service/Generated/Models/SplitSkill.cs

```diff
@@ -0,0 +1,94 @@
+// <auto-generated>
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// </auto-generated>
+
+namespace Microsoft.Azure.Search.Models
+{
+    using Newtonsoft.Json;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    /// <summary>
+    /// The skill to split text.
+    /// <see
+    /// href="https://docs.microsoft.com/azure/search/cognitive-search-skill-textsplit"
+    /// />
+    /// </summary>
+    [Newtonsoft.Json.JsonObject("#Microsoft.Skills.Text.SplitSkill")]
+    public partial class SplitSkill : Skill
+    {
+        /// <summary>
+        /// Initializes a new instance of the SplitSkill class.
+        /// </summary>
+        public SplitSkill()
+        {
+            CustomInit();
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the SplitSkill class.
+        /// </summary>
+        /// <param name="description">The description of the skill which
+        /// describes the inputs, outputs, and usage of the skill.</param>
+        /// <param name="context">The context of the skill.</param>
+        /// <param name="inputs">The inputs of the skill.</param>
+        /// <param name="outputs">The outputs of the skill.</param>
+        /// <param name="defaultLanguageCode">A value indicating which language
+        /// code to use. Default is en.</param>
+        /// <param name="textSplitMode">A value indicating which split mode to
```

Comment:
If `textSplitMode` and `maximumPageLength` are optional, then we should document their default values.

Reply:
Added "x-nullable": false to testSplitMode since it is not optional. Added the default 10000 to maximumPageLength description.